diff --git a/app/src/sources_model.cpp b/app/src/sources_model.cpp
index 08f90a1..241159e 100644
--- a/app/src/sources_model.cpp
+++ b/app/src/sources_model.cpp
@@ -189,7 +189,7 @@ void SourcesModel::reload() noexcept
 
 SourcesModel::Availability SourcesModel::sourceMain() const noexcept
 {
-    return static_cast<SourcesModel::Availability>(m_manager.hasSource(sc::SourceManager::StdSource::Main));
+    return AvailAll;
 }
 
 SourcesModel::Availability SourcesModel::sourceNonFree() const noexcept
