diff --git a/lib/src/file_loader.cpp b/lib/src/file_loader.cpp
index 2170413..9fdd940 100644
--- a/lib/src/file_loader.cpp
+++ b/lib/src/file_loader.cpp
@@ -66,7 +66,7 @@ auto CacheableFileLoader::load(const QString &filename,
 {
     using namespace sc::utils;
 
-#ifdef ENABLE_DEVEL
+//#ifdef ENABLE_DEVEL
     // try load metadata from local dir if devel mode is enabled
     {
         // Try local file
@@ -77,7 +77,7 @@ auto CacheableFileLoader::load(const QString &filename,
             return std::make_pair(std::move(data), sum);
         }
     }
-#endif
+//#endif
 
     // Try to get access to the cache ------------------------------------------
     std::unique_ptr<CacheDb> cachedb;
